src/Simulation/CsharpGeneration/FindNuspecReferences.ps1

```diff
@@ -19,53 +19,52 @@
 # nuget is tracking this problem at: https://github.com/NuGet/Home/issues/4491
 ########################################
```

Comment:
Did you double check the content of the CsharpGeneration NuGet package and compare it against the released one to make sure everything is where it's supposed to be?

Reply:
Seems like everything is the same except for the DLLs and dependencies I added.